packages/lesswrong/server/resolvers/recentDiscussionFeed.ts

```diff
@@ -67,6 +62,23 @@ defineFeedResolver<Date>({
       {[`tagRelevance.${EA_FORUM_TRANSLATION_TOPIC_ID}`]: {$exists: false}},
     ]};
 
+    const postSelector = {
```

Comment:
The changes here are just to separate out shortforms from other types of posts so that we can return the `topLevelComment` with any shortform comments.